app/controllers/users_management/create_users_controller.rb

```diff
@@ -90,7 +91,12 @@ def confirmation
     #    GET /users/set_up
     #
     def set_up
-      # Renders account set up page
+      return handle_missing_invalid_params unless params[:account] && params[:token]
+
+      account = AccountsApi.account(account_id: params[:account])&.symbolize_keys
```

Comment:
I think could be written also as follows (no need to use safe navigator twice):
```suggestion
      account = AccountsApi.account(account_id: params[:account])
      @company_name = account['accountName']&.possessive
```

Reply:
API call returns non-symbolized hash while mocked API call returns symbolized hash. I need to symbolize it to be able to read it in both cases